### PR TITLE
replace string with const

### PR DESCRIPTION
**Pull request by dsy3502**
replace string “KubeEdge”  with const

### CODE REVIEW

**Review by fisherxu — COMMENTED** [review]
@dsy3502 You can add your email addr in the commit to the github settings, or github can't identify your contributions.
image

image

File: common/constants/default.go [hunk 1]
```diff
@@ -14,6 +14,8 @@ const (
 	// NodeName is for the clearer log of cloudcore.
 	NodeName = "NodeName"
 
+	CommonName = "KubeEdge"
```
Comment (gy95) [inline common/constants/default.go:17]: Is this definition OK? @fisherxu @zc2638

Comment (zc2638) [reply common/constants/default.go:17]: like `ProjectName` ? Or other better @gy95

Comment (gy95) [reply common/constants/default.go:17]: yes, this is better. `CommonName` is a little not specific.

Comment (dsy3502) [reply common/constants/default.go]: I have modified it

Comment (gy95) [reply common/constants/default.go]: /LGTM, please squash the commits to one @dsy3502

Comment (dsy3502) [reply common/constants/default.go]: I have fixed it

Comment (gy95) [reply common/constants/default.go]: /lgtm

**Comment by fisherxu** [comment]
> @dsy3502 You can add your email addr in the commit to the github settings, or github can't identify your contributions. image
> 
> image

Would you like to fix this comment? :)

**Comment by dsy3502** [comment]
> 



> > @dsy3502 You can add your email addr in the commit to the github settings, or github can't identify your contributions. image
> > image
> 
> Would you like to fix this comment? :)



> > @dsy3502 You can add your email addr in the commit to the github settings, or github can't identify your contributions. image
> > image
> 
> Would you like to fix this comment? :)

thank you,  now  ok?

**Comment by fisherxu** [comment]
> thank you, now ok?

Not ok :) But I will approve first :) You need to set it via Email column, not in profile.

/approve

**Pull request merged**